userguide/kaui/includes/tags.adoc

```diff
@@ -1 +1,127 @@
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
+==== Types of Tags
+
+Kill Bill has two types of tags:
+
+* *Control tags*&#8212;Also referred to as _system tags_, these tags modify the behavior of the system. In Kaui, you can view and attach these tags, but you cannot add, change, or delete the underlying <<Add Tag Definition, tag definition>>. Kill Bill includes several control tags; see the API documentation for https://killbill.github.io/slate/#tag[a list].
+
+* *User tags*&#8212;These tags are not interpreted by Kill Bill; they are a way for admins or third-party systems to annotate specific existing resources. For instance, the support team could tag account resources associated with a specific customer to group them. In Kaui, you can create and delete user tags on the <<Add Tag Definition, Tag Definition page>>.
+
+[NOTE]
+*Note:* Kill Bill does not include default user tags.
+
+You can view all available tags on the <<View All Tags, Tag Definitions page>>.
+
+==== About Tag Definitions
+
+A tag is based on a _tag definition_. As its name implies, a _tag definition_ defines information about a specific tag. In Kaui, you must create a tag definition for a user tag before you can attach it to an account or invoice.
+
+=== Additional Resources
+
+API Reference:
+
+* https://killbill.github.io/slate/#tag[Tag]
+* https://killbill.github.io/slate/#tag-definition[Tag Definition]
+
+=== View All Tags
+
+To see a list of all available tags in the current tenant, select the tag icon at the top of the screen, then select *Tag Definitions*:
+
+image::../assets/img/kaui/TagDropdown-TagDef.png[width=150,align="center"]
+
+Kaui displays the Tag Definitions page.
+
+image::../assets/img/kaui/TagDefinitionsPage.png[width=850,align="center"]
+
+=== View Objects with Attached Tags
+
+This section explains how to view objects that have attached tags. You can view _all_ the objects with tags for the current tenant; you can also view the tags for a specific account (or invoice).
+
+To see a list of _all_ the objects in the tenant with an attached tag, select the tag icon at the top of the screen, then select *Tags*:
+
+image::../assets/img/kaui/TagDropdown-Tags.png[width=150,align="center"]
+
+Kaui displays the Tags page.
+
+image::../assets/img/kaui/TagsPage-All.png[width=850,align="center"]
+
+[TIP]
+*Tip:* On the Tags page, you can open the object (account, invoice, etc.) by clicking the link in the Object ID column.
+
+To see any tags assigned to a specific account (or invoice):
+
+. Open the account on the Account page.
+. Click *Tags* on the sub-menu.
+
+Kaui opens the Tags page and displays only the tags assigned to that account or its related invoice(s).
+
```

Comment:
Kaui also displays tags associated with subscriptions (if any).

Reply:
I edited the section to indicate the process was the same across accounts, invoices, and subscriptions.